unified_targets/configs/PYRODRONEF4.config

```diff
@@ -0,0 +1,85 @@
+
+board_name PYRODRONEF4
+manufacturer_id PYRO
```

Comment:
This is not an official manufacturer id. Please follow https://github.com/betaflight/betaflight/blob/master/unified_targets/docs/CreatingAUnifiedTarget.md when adding new Unified Target configurations.